dashboard/src/api/resources/Resources.js

```diff
@@ -152,8 +152,17 @@ const SEARCH_URL = {
   apiKey: "",
 };
 
-if (typeof process.env.REACT_APP_API_URL != "undefined") {
+console.log("Node ENV:", process.env.NODE_ENV);
```

Comment:
Do we want to leave these console logs in?

Reply:
good catch, will remove!